pkg/reconciler/v1alpha1/autoscaling/resources/names/names.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package names contains name generating functions for the pod autoscalers.
+package names
+
+// SKSName returns the name of the SKS resource that backs this PA.
+func SKSName(paName string) string {
```

Comment:
nit: `Name` is redundant with the package `names`

Reply:
Done.